packages/analytics/README.md

```diff
@@ -0,0 +1,140 @@
+# `@deriv/analytics`
+
+The analytics package contains all the utility functions used for tracking user events and sending them to the respective platform such as Rudderstack.
+
+**In this document**
+
+-   [RudderStack](#rudderstack)
+    -   [What is RudderStack?](#what-is-rudderstack)
+    -   [Installation](#installation)
+    -   [Usage](#usage)
+    -   [API Reference](#api-reference)
+
+## RudderStack
+
+### What is RudderStack?
+
+RudderStack is a platform that allows you to easily connect different analytical tools and applications such as Google Analytics, and send data and events to the respective tool/application that requires it. Events sent to RudderStack will be routed to the respective tool and application, similar to a reverse proxy but for data and analytics.
+
+### Installation
+
+To start tracking with Rudderstack, first add it as a dependency in `package.json`:
+
+```js
+{
+    ...
+    "dependencies": {
+        ...
+        "@deriv/analytics": "^1.0.0",
+        ...
+    }
+    ...
+}
+```
+
+Then run `npm run bootstrap` as well as `npm run build:all` to install the dependencies and build it.
```

Comment:
Shouldn't this be like this?
```suggestion
Then run `npm run bootstrap:dev` as well as `npm run build:all` to install the dependencies and build it.
```

Reply:
Sure, I'll update the docs to include this